runtime/src/rent_collector.rs

```diff
@@ -52,6 +58,36 @@ impl RentCollector {
         }
     }
 
+    /// true if it is easy to determine this account should not have rent collected from it
+    pub fn no_rent(
```

Comment:
pet peeve: Can this function be renamed so it's is not a negative? Something like "should_collect_rent" maybe?

Reply:
ok, I'll pick that up in a next pr.